optionals/auto_load_settings_file/loadSettingsFile.sqf

```diff
@@ -0,0 +1,14 @@
+
+if (isFilePatchingEnabled) then {
+    private _source = ["client", "server"] select (isMultiplayer && isServer);
+    private _file = loadFile PATH_SETTINGS_FILE;
+
+    if (_file != "") then {
+        [_file, _source] call FUNC(import);
```

Comment:
`EFUNC(settings,import)`

and import calls `FUNC(gui_refresh)` which won't work on a dedicated or a client that hasn't opened the menu
maybe a call to `CBA_settings_fnc_set` with an extra flag that doesn't change the profileNamespace setting?